target/linux/qualcommax/ipq50xx/base-files/lib/upgrade/glinet-upgrade.sh

```diff
@@ -0,0 +1,53 @@
+. /lib/upgrade/common.sh
+
+glinet_do_fit_upgrade() {
+	echo -n "fit: Extract [ FIT IMAGE ] -x-x-> [ ubi.bin ] ... "
+	local ubi=/tmp/ubi.bin
+	local part=$(dumpimage -l "$1" | grep -o "Image [0-9] (ubi)" | cut -f2 -d" ")
+
+	local ubibin=$( dumpimage -T flat_dt -p ${part} -o "$ubi"  $1 )
```

Comment:
There are a few missing quotes, and you don't need the `ubibin variable:
```suggestion
	dumpimage -T flat_dt -p "$part" -o "$ubi" "$1"
```